packages/trip-details/package.json

```diff
@@ -9,20 +9,21 @@
   "author": "Evan Siroky",
   "license": "MIT",
   "private": false,
+  "files": [
+    "i18n"
+  ],
```

Comment:
This might result in only this folder and not the `lib` or `esm` folders being included in the release. What was the intention of putting this in here? Perhaps there is a different way to accomplish that.

Reply:
Yeah that was not necessary. Removed in https://github.com/opentripplanner/otp-ui/pull/281/commits/51d8398e9227f98c910f7b20db5363a707c5f97a.